features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -333,3 +333,23 @@
   step %{I open signups for "Awesome Gift Exchange"}
   step %{everyone has signed up for the gift exchange "Awesome Gift Exchange"}
 end
+
+### Deleting things
+# use the same names as "everyone is signed up"
+
+Then /^no one should be signed up for "([^\"]*)"$/ do |challenge|
+  collection = Collection.find_by_title(challenge)
+  %w(myname1 myname2 myname3 myname4).each do |name|
```

Comment:
No, we just need to check the users who have signed up in "Given everyone has signed up for the gift exchange Bad Gift Exchange":
```
Given /^everyone has signed up for the gift exchange "([^\"]*)"$/ do |challengename|
  step %{I am logged in as "myname1"}
  step %{I sign up for "#{challengename}" with combination A}
  step %{I am logged in as "myname2"}
  step %{I sign up for "#{challengename}" with combination B}
  step %{I am logged in as "myname3"}
  step %{I sign up for "#{challengename}" with combination C}
  step %{I am logged in as "myname4"}
  step %{I sign up for "#{challengename}" with combination D}
end
```